### PR TITLE
Bug 1133402 - Add a script to optimise SVG images

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=87d3fe15b7a6)

File: tools/svg_recompress.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# Option variables
+
+opt_help=no
+opt_paths=""
+
+###############################################################################
+# Print the help message
+
+print_help() {
+    printf "Usage: `basename $0` [PATH...]
```
Comment (gabrielesvelto) [inline tools/svg_recompress.sh]: nit: I would add `[OPTION...]` before `[PATH...]` for consistency but it's just me being pedantic :-)

Comment (gmarty) [inline tools/svg_recompress.sh]: That's correct! I updated the patch. Thanks for being pedantic :-)

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5cf2c0ec41fa)

**Pull request merged**